tailwind.config.js

```diff
@@ -0,0 +1,706 @@
+module.exports = {
```

Comment:
This seems very large. Have you customised all of this, or is this the defaults? My tailwind.config.js is much smaller.... 😁

Reply:
nope! I just took the WHOLE default, I'm trying out tailwind for the first time... It's a valid point though!

Reply:
adding a purge item to the tailwind config based on this: 
https://tailwindcss.com/docs/controlling-file-size/